src/math/ceil.js

```diff
@@ -0,0 +1,35 @@
+const ceil = (number, precision) => {
+    let numberString = number.toString();
+    let decimalIndex = numberString.indexOf('.');
+    let beforeDecimal = numberString.substring(0, decimalIndex);
+    let afterDecimal = numberString.substring(decimalIndex, numberString.length);
+  if (Number.isInteger(number)){
+    return number;
+  } else if (Number.isInteger(number) === false) {
```

Comment:
This doesn't need to be an else if, it can just be and else without a predicate check.

ie.
```
if( Number.isInteger(number) === false ) {
  //do the things

} else {
  // all the other things

}